### PR TITLE
fix async model load

### PR DESCRIPTION
**Pull request by paulsouche**

### CODE REVIEW

**Comment by cbaur** [comment]
Please merge this! It just fixed a huge problem in our app. Thank you, Paul!

File: dist/ng-sortable.js [hunk 1]
```diff
@@ -814,6 +810,7 @@
                 }
               }
               if (targetScope.type === 'sortable') {//sortable scope.
+                targetElement = angular.element(placeElement).parent();
```
Comment (connec) [inline dist/ng-sortable.js]: Why was this line changed?

Comment (paulsouche) [inline dist/ng-sortable.js]: Because I forked this Repo before #149 and the dist contained this line.

Comment (paulsouche) [inline dist/ng-sortable.js]: updated

**Comment by connec** [comment]
I also ended up making this change on my own fork.  It would be great if it could be merged.

**Comment by a5hik** [comment]
Ok let me do it, and test.
